src/Components/Notes/NoteCard.jsx

```diff
@@ -169,16 +165,26 @@ export default function NoteCard({
     setComments(newComments)
   } */
 
-
   /** Update issue on GH, set read-only */
-  async function submitUpdate() {
+  async function updateIssueGithub() {
     const res = await updateIssue(repository, noteNumber, title, editBody, accessToken)
     const editedNote = notes.find((note) => note.id === id)
     editedNote.body = res.data.body
     setNotes(notes)
     setEditMode(false)
   }
 
+  /**
+   * Update comment in github
+   *
+   * @param {number} commentId
+   */
+    async function updateCommentGithub(commentId) {
+      const updatedComment = await updateComment(repository, commentId, editBody, accessToken)
```

Comment:
Fix this?  https://github.com/bldrs-ai/Share/issues/1201

Reply:
This is a bit more involved.
It was captured in a separate ticket, so I will take care of it in s separate PR.